actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -241,6 +246,18 @@ public void setMode(final StageMode mode)
         hosting.start();
         execution.start();
 
+        try
+        {
+            Class.forName("com.ea.orbit.metrics.MetricsManager"); //make sure the metrics manager is on the classpath.
+
+            MetricsManager.getInstance().initializeMetrics(metricsConfig);
```

Comment:
Unfortunately this isn't going to work, we cannot have imports for MetricsManager or reference it directly if you want the dependency to be optional, instead we'll need to use reflection to instantiate the manager.